### PR TITLE
Replace Ritie with Rita

### PR DESCRIPTION
**Pull request by Nyx102**

### CODE REVIEW

File: Volumes/Volume 3/Text/1.md [hunk 1]
```diff
@@ -112,4 +112,4 @@ Naturally, the battle has long been settled.
 
 It goes without saying, but the defeated girl lost her life. Likewise, the victor perished shortly after the duel. Each closed their eyes without seeing the future they wished for.
 
-There is hardly anyone left who even knows that this battle happened.
+There is hardly anyone left who even knows that this battle happened.
```
Comment (WorldEnd-Formatting) [inline Volumes/Volume 3/Text/1.md:115]: What actually changed on this line? I'm having trouble seeing it

**Comment by WorldEnd-Admin** [comment]
GitHub is showing a bunch of whitespace-only changes to files that don't involve Rita/Ritie. Would you mind just double checking that those are supposed to be changing?

**Comment by Nyx102** [comment]
It's just that I removed some newline characters at the end of the files for consistency, but I can get rid of those if you want. Since I realize now that some of the other files also have a newline at the end.

**Comment by WorldEnd-Admin** [comment]
Ah I see. If it's just newlines at the end it shouldn't matter, at least for the TeX, since it strips whitespace from the beginning and end of each paragraph.

I've merged, but feel free to change newlines in any future PR, just leave a comment so I know what's changing since GitHub doesn't make it super clear.

**Pull request merged**